services/movie.py

```diff
@@ -6,9 +7,13 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
 ) -> QuerySet:
```

Comment:
To improve code clarity, it’s best to specify the type of `QuerySet` returned in the type annotations.
```suggestion
) -> QuerySet [Movie]:
```